exonum/src/storage/proof_map_index/key.rs

```diff
@@ -259,52 +263,99 @@ impl PartialEq for DBKey {
 
 impl ::std::fmt::Debug for DBKey {
     fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        write!(f, "DBKey(")?;
-        for i in 0..self.len() {
-            write!(
-                f,
-                "{}",
-                match self.get(i) {
-                    ChildKind::Left => '0',
-                    ChildKind::Right => '1',
+        let mut bits = String::with_capacity(KEY_SIZE * 8);
+        for byte in 0..self.data.len() {
+            let chunk = self.data[byte];
+            for bit in (0..8).rev() {
+                let i = (byte * 8 + bit) as u16;
+                match i {
+                    _ if i < self.from => write!(&mut bits, "_")?,
+                    _ if i >= self.to => write!(&mut bits, "_")?,
+                    _ => {
+                        write!(
+                            &mut bits,
+                            "{}",
+                            if (1 << bit) & chunk == 0 { '0' } else { '1' }
+                        )?
```

Comment:
Nit: it is possible to reduce strength from `write!`  to [push](https://doc.rust-lang.org/std/string/struct.String.html?search=#method.push)